src/main/kotlin/edu/byu/mealplanningassistant/service/RecipeService.kt

```diff
@@ -24,7 +24,13 @@ class RecipeService(val db: RecipesRepository) {
         val lunch = db.getRecipesWithTags((request.requests[0].tags.toHashSet() + hashSetOf("lunch")) as HashSet<String>)
         val dinner = db.getRecipesWithTags((request.requests[0].tags.toHashSet() + hashSetOf("dinner")) as HashSet<String>)
 
-        return GetRandomizedRecipeBatchResponse(breakfast, lunch, dinner, true, "")
+        val bld = listOf(breakfast, lunch, dinner).map { recipeList -> recipeList.filter { recipe ->
```

Comment:
I think this is a great start but I think we want to make sure we're still returning the right number of recipes right? Also I'm not sure you have the most recent version of the code because I have lots of logic to make sure we have the right tags and return the right number of unique recipes.